docs/reference/commandline/system_events.md

```diff
@@ -0,0 +1,349 @@
+---
+title: "system events"
+description: "The system events command description and usage"
+keywords: "system, events, container, report"
+---
+
+<!-- This file is maintained within the moby/moby Github
+     repository at https://github.com/moby/moby/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# system events
+
+```markdown
+Usage:  docker system events [OPTIONS]
+
+Get real time events from the server
+
+Options:
+  -f, --filter value   Filter output based on conditions provided (default [])
+      --format string  Format the output using the given Go template
+      --help           Print usage
+      --since string   Show all events created since timestamp
+      --until string   Stream events until this timestamp
+```
+
+## Description
+
+Use `docker system events` to get real-time events from the server. These
+events differ per Docker object type.
+
+### Object types
+
+#### Containers
+
+Docker containers report the following events:
+
+- `attach`
+- `commit`
+- `copy`
+- `create`
+- `destroy`
+- `detach`
+- `die`
+- `exec_create`
+- `exec_detach`
+- `exec_start`
+- `export`
+- `health_status`
+- `kill`
+- `oom`
+- `pause`
+- `rename`
+- `resize`
+- `restart`
+- `start`
+- `stop`
+- `top`
+- `unpause`
+- `update`
+
+#### Images
+
+Docker images report the following events:
+
+- `delete`
+- `import`
+- `load`
+- `pull`
+- `push`
+- `save`
+- `tag`
+- `untag`
+
+#### Plugins
+
+Docker plugins report the following events:
+
+- `install`
+- `enable`
+- `disable`
+- `remove`
+
+#### Volumes
+
+Docker volumes report the following events:
+
+- `create`
+- `mount`
+- `unmount`
+- `destroy`
+
+#### Networks
+
+Docker networks report the following events:
+
+- `create`
+- `connect`
+- `disconnect`
+- `destroy`
+
+#### Daemons
+
+Docker daemons report the following events:
+
+- `reload`
+
```

Comment:
Sory gut can you please undo this change? When I wrote the comment, I didn't realize that you had an Examples header below.

Also, for this repository, you need to squash your commits down into one before final review. To do that, issue the command `git rebase -i master`. An editor (probably vi) will open, change the lines for all the commits after the first one to say `squash` instead of `pick`, then save. You will get a chance to edit the combined commit messages. Make sure you don't remove the `Signed-off by` line. Save that editor, then you will end up with only one commit. Then to get it into the pull request, you need to do a force push `git push -f`.